src/commands/support/reportedissues.ts

```diff
@@ -1,17 +1,83 @@
+import { Colors, EmbedField } from 'discord.js';
 import { CommandDefinition } from '../../lib/command';
 import { CommandCategory } from '../../constants';
 import { makeEmbed } from '../../lib/embed';
+import Logger from '../../lib/logger';
+
+const jsdom = require('jsdom');
+
+const { JSDOM } = jsdom;
+
+const FBW_DOCS_REPORTED_ISSUES_URL = 'https://docs.flybywiresim.com/fbw-a32nx/support/reported-issues/';
+
+const genericReportedIssuesEmbed = makeEmbed({
+    title: 'FlyByWire A32NX | Reported Issues',
+    description: `Please see [this link](${FBW_DOCS_REPORTED_ISSUES_URL}) for a current list of reported issues.`,
+});
+
+const foundIssuesEmbed = (fields: EmbedField[]) => makeEmbed({
+    title: 'FlyByWire A32NX | Reported Issues',
+    description: 'Your issue is in our reported issues list. Please try the possible solutions in the following link(s) and report back if it didn\'t help, with all the steps you tried.',
+    fields,
+});
+
+const foundIssueEmbedFields = (id: string): EmbedField[] => [
+    {
+        inline: false,
+        name: `${id.replace(/-/gi, ' ')}`,
+        value: `[link to reported issue section](${FBW_DOCS_REPORTED_ISSUES_URL}#${id})`,
+    },
+];
+
+const notFoundEmbed = (action: string, command: string) => makeEmbed({
+    title: `Reported Issues - ${action} - ${command} not found`,
```

Comment:
```suggestion
const notFoundEmbed = (searchArgs: string) => makeEmbed({
    title: `Reported Issues - ${searchArgs} not found`,
```

There's no sub commands anymore, so having "Show" show up isn't necessary, and renamed the variable to be more clear.